apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/BrevDataMapper.kt

```diff
@@ -199,19 +206,26 @@ class BrevDataMapper(
                     val etterbetaling = async { fetcher.hentEtterbetaling() }
                     val trygdetid = async { fetcher.hentTrygdetid() }
```

Comment:
Er ikkje heilt fan av dette oppsettet, men har ikkje fått a-ha-augneblinken for ei betre løysing enno. Kjem vel plutseleg.

Reply:
Enig at dette ikke er optimalt, men lar det være inntil videre